src/styles/StyleUtils.js

```diff
@@ -355,6 +355,18 @@ function getBackgroundColorStyle(backgroundColor) {
     };
 }
 
+/**
+ * Returns a style for text color
+ *
+ * @param {String} color
+ * @returns {Object}
+ */
+function getTextColorStyle(color) {
```

Comment:
Why do we need this vs just using `color={colors.blueLinkPreview}`?

Reply:
@thesahindia pointed it out in the previous comment https://github.com/Expensify/App/pull/21717#discussion_r1245383782

> You will need to pass the color in style prop because we don't have the color prop in `TextLink` component

Reply:
Nice, thanks for clarifying!